eth_keys/datatypes.py

```diff
@@ -57,163 +58,195 @@
     ByteString = collections.abc.ByteString
 
 
-class BackendProxied(object):
-    _backend = None
+class LazyBackend:
+    def __init__(self, backend=None):
+        # type: (Union[BaseECCBackend, Type[BaseECCBackend], str, None]) -> None
```

Comment:
Any reason to keep this annotation as a comment or just an oversight?

Reply:
Preventing circular imports if I recall correctly.

Reply:
To workaround circular imports there's this trick: http://mypy.readthedocs.io/en/latest/common_issues.html#import-cycles